src/applications/edu-benefits/10203/containers/IntroductionPage.jsx

```diff
@@ -1,17 +1,44 @@
 import React from 'react';
-import { focusElement } from 'platform/utilities/ui';
+
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { focusElement } from '@department-of-veterans-affairs/platform-utilities/ui';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
-import { connect } from 'react-redux';
 import { getRemainingEntitlement } from '../actions/post-911-gib-status';
+import { useFeatureToggle } from '~/platform/utilities/feature-toggles';
+
+const withFeatureToggle = Component => {
+  return props => {
+    const {
+      useToggleValue,
+      useToggleLoadingValue,
+      TOGGLE_NAMES,
+    } = useFeatureToggle();
+
+    const toggleValue = useToggleValue(
+      TOGGLE_NAMES.benefitsEducationUseLighthouse,
+    );
+    const togglesLoading = useToggleLoadingValue();
+
+    if (togglesLoading) {
+      return null;
+    }
+
+    const apiVersion = { apiVersion: toggleValue ? 'v1' : 'v0' };
+
+    return <Component {...props} apiVersion={apiVersion} />;
+  };
+};
```

Comment:
HOC to wrap the class-based component and allow use of the hook to look at feature toggle